cmd/geth/governancedeploy_test.go

```diff
@@ -0,0 +1,114 @@
+// governancedeploy.js
+
+package main
+
+import (
+	"encoding/json"
+	"math"
+	"math/big"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind/backends"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/params"
+	gov "github.com/ethereum/go-ethereum/wemix/bind"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeployGoverananceContracts(t *testing.T) {
+	owner, err := crypto.GenerateKey()
+	require.NoError(t, err)
+
+	opts, err := bind.NewKeyedTransactorWithChainID(owner, params.AllEthashProtocolChanges.ChainID)
+	require.NoError(t, err)
+
+	backend := backends.NewSimulatedBackend(
+		core.GenesisAlloc{
+			opts.From: {Balance: new(big.Int).Mul(big.NewInt(math.MaxInt64), big.NewInt(params.Ether))},
+		},
+		params.MaxGasLimit,
+	)
+
+	// make temp config.json
+	configJSFile := filepath.Join(t.TempDir(), "config.json")
+	require.NoError(t, os.WriteFile(configJSFile, []byte(configStr), 0775))
+
+	go func() {
+		for {
+			time.Sleep(0.5e9)
+			backend.Commit()
+		}
+	}()
+
+	domains, env, members, err := getInitialGovernanceInitDatas(configJSFile)
+	require.NoError(t, err)
+	for name, addr := range domains {
+		t.Log(name, addr)
+	}
+	bytes, _ := json.MarshalIndent(env, "", "  ")
+	t.Log(string(bytes))
+	for i, member := range members {
+		t.Log(i, member)
+	}
+	require.NoError(t, deployGovernance(backend, opts, gov.DefaultInitEnvStorage.STAKING_MIN, configJSFile))
```

Comment:
detailed tick processing

```suggestion
	fin := make(chan struct{})
	nonce, err := backend.NonceAt(context.Background(), opts.From, nil)
	require.NoError(t, err)

	go func() {
		ticker := time.NewTicker(0.1e9)
		defer ticker.Stop()

		for {
			select {
			case <-ticker.C:
				pending, err := backend.PendingNonceAt(context.Background(), opts.From)
				require.NoError(t, err)

				if nonce != pending {
					nonce = pending
					backend.Commit()
				}
			case <-fin:
				return
			}

		}
	}()

	domains, env, members, err := getInitialGovernanceInitDatas(configJSFile)
	require.NoError(t, err)
	for name, addr := range domains {
		t.Log(name, addr)
	}
	bytes, _ := json.MarshalIndent(env, "", "  ")
	t.Log(string(bytes))
	for i, member := range members {
		t.Log(i, member)
	}
	require.NoError(t, deployGovernance(backend, opts, gov.DefaultInitEnvStorage.STAKING_MIN, configJSFile))
	close(fin)
```